packages/react-magma-dom/src/hooks/useFocusLock.ts

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
```

Comment:
should we create a docs page for this? Makes sense that other teams (or components external to dom) would want this too

Reply:
in the same vein... we should export it in `./src/index.ts`